tests/unit/pillar_test.py

```diff
@@ -130,38 +130,6 @@ def test_topfile_order(self, Matcher, get_file_client):
         pillar = salt.pillar.Pillar(opts, grains, 'mocked-minion', 'base')
         self.assertEqual(pillar.compile_pillar()['ssh'], 'foo')
 
-    @patch('salt.pillar.salt.fileclient.get_file_client', autospec=True)
```

Comment:
They were dropped because there'd be no pillar_source_merging_strategy of recurse_list as it would no longer be necessary.